src/plugins/intel_gpu/src/plugin/transformations_pipeline.cpp

```diff
@@ -907,6 +912,52 @@ void TransformationsPipeline::apply(std::shared_ptr<ov::Model> func) {
         // not working properly.
         manager.register_pass<ov::pass::Validate>();
 
+        float activations_scale_factor = config.get_property(ov::hint::activations_scale_factor);
+        ov::element::Type scaled_precision = element::f16;
+
+        if (activations_scale_factor > 0.f) {
+            using namespace ov::pass::low_precision;
+
+            auto supportedPrecisions = std::vector<PrecisionsRestriction>({});
+            auto perTensorQuantization = std::vector<QuantizationGranularityRestriction>({});
+
+            auto pass_config = manager.get_pass_config();
+            pass_config->disable<ov::pass::AddMultiplyFusion>();
+            pass_config->disable<RecurrentCellTransformation>();
+            pass_config->disable<MultiplyToGroupConvolutionTransformation>();
+            pass_config->disable<ConvolutionTransformation>();
+            pass_config->disable<ConvolutionBackpropDataTransformation>();
+            pass_config->disable<GroupConvolutionTransformation>();
+            pass_config->disable<MatMulTransformation>();
+            pass_config->disable<MVNTransformation>();
+            pass_config->disable<ConcatTransformation>();
+
+            pass_config->set_callback<FoldConvertTransformation>(
+                [](const std::shared_ptr<const ov::Node> &node) -> bool {
+                    return ov::is_dequantization_node(node);
+                });
+
+            pass_config->set_callback<FuseConvertTransformation>(
+                [](const std::shared_ptr<const ov::Node> &node) -> bool {
+                    return (ov::is_dequantization_node(node) || ov::is_type<ov::opset1::FakeQuantize>(node));
+                });
+
+            manager.register_pass<ov::pass::activations_scaling::ScaleDownSingleLayer>(activations_scale_factor, scaled_precision);
+            manager.register_pass<ov::pass::activations_scaling::ScaleDownFusion>();
+            auto params = LayerTransformation::Params(false, scaled_precision, {scaled_precision}, true, false);
+            auto lpt_pass = manager.register_pass<LowPrecision>(supportedPrecisions, perTensorQuantization, params);
+            lpt_pass->add_main<ov::pass::activations_scaling::MulGroupNormTransformation>();
```

Comment:
We also need a pass to handle SDPA op. Otherwise we'll have explicit Multiply op for QKV inputs and transpose won't be fused